### PR TITLE
feat: SVGO config

### PR DESCRIPTION
**Pull request by dlnr**
# Describe the pull request

In preparation for new iconset.

## What

SVGO config

## Why

Easier for other developers to also export iconset when they are done.

## Checklist

Before submitting your pull request, please ensure you have done the following. Check each checkmark if you have done so or if it wasn't necessary:

- [X] Add or update documentation
- [X] Start the PR title with a Conventional Commit prefix, [as explained here](https://github.com/Amsterdam/design-system/blob/main/documentation/publishing.md?plain=1#L11).

## Additional notes

- Provide links to any related issues or discussions.
- Add a link to the specific story in the feature branch deploy.
- Mention any areas where additional review or feedback is needed.

### CODE REVIEW

File: proprietary/assets/README.md [hunk 1]
```diff
@@ -8,3 +8,11 @@ Assets for the City of Amsterdam
...
 See the Design System documentation for guides on [the web manifest and application icons](https://designsystem.amsterdam/?path=/docs/docs-developer-guide-web-app--docs)
 and [the favicon](https://designsystem.amsterdam/?path=/docs/brand-assets-favicon--docs).
+
+## Exporting icons
+
+To export all icons from Figma to SVG use the native export functionality in Figma. To select all icons select the frame, press `Enter` to select direct children and then `Enter` again to select all icon components. The select export to SVG in the sidebar and export all icons to the `icons` folder. Then run the following command to optimize the SVG files:
+
+```bash
+npm run optimize-icons
+```
```
[the remark this answers is not in the document]
Comment (dlnr) [reply proprietary/assets/README.md:18]: This was targeted at developers who work on this after me.

File: proprietary/react-icons/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Contents
+
+- [Exporting icons](#exporting-icons)
+
+## Exporting icons
+
+To export all icons from [the Figma file](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=6852-5124) to SVG use the native SVG export functionality in Figma. To select all icons select the shapes frame, press `Enter` to select direct children and then `Enter` again to select all icon components. The select export to SVG in the sidebar and export all icons to the `icons` folder. Then run the following command to optimize the SVG files:
+
+```bash
```
Comment (VincentSmedinga) [inline proprietary/react-icons/CONTRIBUTING.md:13]: I shortened this when updating the readmes.

```suggestion
```sh
```

File: proprietary/react-icons/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# Contributing Guidelines
+
+## Contents
+
+- [Exporting icons](#exporting-icons)
+
+## Exporting icons
+
+To export all icons from [the Figma file](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=6852-5124) to SVG use the native SVG export functionality in Figma. To select all icons select the shapes frame, press `Enter` to select direct children and then `Enter` again to select all icon components. The select export to SVG in the sidebar and export all icons to the `icons` folder. Then run the following command to optimize the SVG files:
```
Comment (alimpens) [inline proprietary/react-icons/CONTRIBUTING.md:11]:
```suggestion
## Importing icons from Figma
We create and update icons in Figma, then import them into this package for distribution.
**Important**: Only export icons from Figma for use in this package. Do not use these icons directly in an application.
To export all icons from [the Figma file](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=6852-5124), use Figma's built-in SVG export feature.
Select the Shape frame, press `Enter` to select direct children and then press `Enter` again to select all icon components.
In the right-hand sidebar, click the plus sign next to Export. Choose SVG and export.
Copy these files over to the `icons` folder and run the following command to optimize them:
```

File: proprietary/react-icons/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!-- @license CC0-1.0 -->
```
Comment (alimpens) [inline proprietary/react-icons/CONTRIBUTING.md:1]: I think this file is in the wrong location, it should be in `proprietary/assets`

File: proprietary/react-icons/CONTRIBUTING.md (hunk 3 above)
Comment (dlnr) [reply proprietary/react-icons/CONTRIBUTING.md:11]: This is not correct, I've described how you can export multiple files, you've changed it to selecting the shape which has no children.

File: proprietary/react-icons/CONTRIBUTING.md (hunk 4 above)
Comment (dlnr) [reply proprietary/react-icons/CONTRIBUTING.md]: Moved it

File: proprietary/assets/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<!-- @license CC0-1.0 -->
+
+# Contributing Guidelines
+
+## Importing icons from Figma
+
+We create and update icons in Figma, then import them into this package for distribution.
+**Important**: Only export icons from Figma for use in this package. Do not use these icons directly in an application.
```
Comment (alimpens) [inline proprietary/assets/CONTRIBUTING.md:8]:
```suggestion

**Important**: Only export icons from Figma for use in this package. Do not use these icons directly in an application.

```

**Review by alimpens — APPROVED** [review]

**Pull request merged**